docs/reference/mapping/params/ignore-malformed.asciidoc

```diff
@@ -49,6 +49,7 @@ PUT my-index-000001/_doc/2
 The `ignore_malformed` setting is currently supported by the following <<mapping-types,mapping types>>:
 
 <<number>>::         `long`, `integer`, `short`, `byte`, `double`, `float`, `half_float`, `scaled_float`
+<<boolean>>::        `true`, `false`
```

Comment:
Here the list following the type shouldn't be all accepted values but all subtypes that support zhe setting. In this case there is only `boolean`

Reply:
That means you are asking to replace ``` `true`, `false` ``` with ``` `boolean` ```, Right?

Reply:
Changed it to ``` `boolean` ```.